frontends/web/src/routes/account/summary/chart.tsx

```diff
@@ -23,9 +23,10 @@ import { Amount } from '../../../components/amount/amount';
 import { PercentageDiff } from './percentage-diff';
 import Filters from './filters';
 import { getDarkmode } from '../../../components/darkmode/darkmode';
-import { TChartDisplay, TChartFiltersProps } from './types';
+import { TChartFiltersProps } from './types';
```

Comment:
since `TChartDisplay` isn't used in Chart component, maybe move the type to appprovider?